app/src/main/java/com/example/springprojectstationbuild/dto/ComputerComponent.kt

```diff
@@ -2,9 +2,12 @@ package com.example.springprojectstationbuild.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class ComputerComponent(@SerializedName("Price") var price : Int, @SerializedName("Name") var name: String){
-    private var component = name + " " + price
-    override fun toString(): String {
+data class ComputerComponent(
+    @SerializedName("Price") var price : Int,
+    @SerializedName("Name") var name: String
+    ){
+        var component = name + " " + price
+        override fun toString(): String {
```

Comment:
According to me from https://kotlinlang.org/docs/reference/coding-conventions.html The function signature doesn't fit on a single line, so I tried to modify it